docs/source/tutorials/function_print_stat.ipynb

```diff
@@ -0,0 +1,220 @@
+{
```

Comment:
I'd like to see a test printing all kind of objects directly. Right now you are printing an `AggregatedResult` in all the examples in here so it'd be nice to print directly the other types there (i.e. `print_stats(results["my_device"])` and `print_stats(results["my_device"][0])`

Reply:
Added `MultiResult` and `Result` objects to `function_print_stat.ipynb` tutorial